### DISCUSSION TITLE
Multiple Dials Mapped?

### DISCUSSION THREAD

**Discussion by Eeostatic** [WhiteMagic/JoystickGremlin#583] [General]
I bought this secondary key pad thing to see if it would work for some Star Citizen. All the inputs function but I cant get the dials to register with Gremlin as dials or sliders. They only register and can be remapped to buttons. Am I missing something or is this function not possible?
![PXL_20241217_234246671 MP](https://github.com/user-attachments/assets/d212d3dd-b742-44dc-a146-b8f19f670f05)

**Comment by WhiteMagic** [comment]
Without knowing how they are exposed to DirectInput I can't say if that behaviour is a bug or actually just a device limitation. What do the dials show up as in Window's joy.cpl tool?
